lib/elixir_sense/providers/suggestion.ex

```diff
@@ -147,21 +147,16 @@ defmodule ElixirSense.Providers.Suggestion do
     end
   end
 
-  @spec find_hint_mods_funcs(String.t, [module], [{module, module}]) :: %{hint: hint, suggestions: [mod | func]}
-  defp find_hint_mods_funcs(hint, imports, aliases) do
-    Application.put_env(:"alchemist.el", :aliases, aliases)
-
-    list1 = Complete.run(hint, imports)
```

Comment:
I'm pretty sure this is no longer needed as the call `Complete.run(hint, env)` covers imports and local module calls

Reply:
I think the `Complete` module will eventually need to be refactored or even replaced. It's more complex than it should especially because of the hint stuff, which is a nice feature for `iex` but I'm not sure it's relevant for editors.